modules/dedicated-configuring-your-application-routes.adoc

```diff
@@ -5,47 +5,29 @@
-
-If you want to use custom domain names for your application routes, {product-title} supports
-CNAME records in your DNS configuration that point to
-`elb.apps.<cluster-id>.<shard-id>.p1.openshiftapps.com`. While `elb` is recommended as a
-reminder for where this record is pointing, you can use any string for this
-value. You can create these CNAME records for each custom route you have, or you
-can create a wildcard CNAME record. For example:
+When your cluster is provisioned, an AWS elastic load balancer (ELB) is created to route application traffic into the cluster. The domain for your ELB is configured to route application traffic via `http(s)://*.<cluster-id>.<shard-id>.p1.openshiftapps.com`. The `<shard-id>` is a random four-character string assigned to your cluster at creation time.
 
+If you want to use custom domain names for your application routes, {product-title} supports CNAME records in your DNS configuration that point to `elb.apps.<cluster-id>.<shard-id>.p1.openshiftapps.com`. While `elb` is recommended as a reminder for where this record is pointing, you can use any string for this value. You can create these CNAME records for each custom route you have, or you can create a wildcard CNAME record. For example:
+
+[source,yaml]
 ----
 *.openshift.example.com    CNAME    elb.apps.my-example.a1b2.p1.openshiftapps.com
 ----
 
-This allows you to create routes like *_app1.openshift.example.com_* and
-*_app2.openshift.example.com_* without having to update your DNS every time.
-
-////
+This allows you to create routes like `app1.openshift.example.com` and `app2.openshift.example.com` without having to update your DNS every time.
 
-Customers with configured VPC peering or VPN connections have the option of
-requesting a second ELB, so that application routes can be configured as
-internal-only or externally available. The domain for this ELB will be identical
-to the first, with a different `<shard-id>` value. By default, application
-routes are handled by the internal-only router. To expose an application or
-service externally, you must create a new route with a specific label,
-`route=external`.
+Customers with configured VPC peering or VPN connections have the option of requesting a second ELB, so that application routes can be configured as internal-only or externally available. The domain for this ELB will be identical to the first, with a different `<shard-id>` value. By default, application routes are handled by the internal-only router. To expose an application or service externally, you must create a new route with a specific label, `route=external`.
 
-To expose a new route for an existing service, apply the label `route=external`
-and define a host name that contains the secondary, public router shard ID:
+To expose a new route for an existing service, apply the label `route=external` and define a host name that contains the secondary, public router shard ID:
```

Comment:
```suggestion
Using our recommendation, in order to expose a new, external route for an existing service, apply the label `route=external` and manually define a host name that uses the `apps2` domain instead of the default `apps` domain:
```